lib/db/mysql.js

```diff
@@ -1460,7 +1459,7 @@ module.exports = function (log, error) {
     // recovery codes is done is two separate procedures. First one
     // deletes all current codes and the second one inserts the
     // hashed randomly generated codes.
-    return dbUtil.generateRecoveryCodes(count, RECOVERY_CODE_KEYSPACE, RECOVERY_CODE_LENGTH)
+    return dbUtil.generateRecoveryCodes(count, RECOVERY_CODE_LENGTH)
       .then((codes) => {
         return this.read(DELETE_RECOVERY_CODES, [uid])
           .then(() => codes.map((code) => dbUtil.createHashScrypt(code)))
```

Comment:
@deeptibaghel Let's lowercase the code here and add a comment that this was done for backwards comparability  and because content-server normalizes them to lowercase.